components/patient-data-sharing/push-api/src/main/resources/ApplicationResources.properties

```diff
@@ -48,27 +49,27 @@ phenotips.PushPatient.male=Male
 phenotips.PushPatient.unknown=Unknown
 phenotips.PushPatient.negative=NO
 phenotips.PushPatient.oneServerMessage=There is only one configured remote server: 
-phenotips.PushPatient.patientUploadedMessage=This patient has been uploaded to this server
+phenotips.PushPatient.patientUploadedMessage=This patient was uploaded to
 phenotips.PushPatient.patientUploadedLessThanHour=less than an hour ago
 phenotips.PushPatient.patientUploadedDays=days ago
 phenotips.PushPatient.patientUploadedHours=hours ago
-phenotips.PushPatient.patientUploadedAsID=as
-phenotips.PushPatient.patientUploadedUpdate=Further pushes will attempt to update the patient on the remote server.
+phenotips.PushPatient.patientUploadedAsID=and was assigned patient ID
+phenotips.PushPatient.patientUploadedUpdate=If data in the record has changed, push to __serverID__ again to update the record there.
```

Comment:
BTW, I think __description_of_expected_variable__ (in some way or form) is more useful than {0} (including for human translators, might not immediately know what is supposed to go in place of {0} in all sentences, and maybe should use a different article or what not in whatever language, depending on the exact expected meaning of the inserted part)